From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Tue, 25 Jul 2017 17:34:31 +0200
Subject: [PATCH] resource: Fix ovirt_resource_rest_call_sync return value

Upon success, ovirt_resource_rest_call_sync() was always returning
NULL, which is not what is expected. This, among other things, made
ovirt_resource_refresh() non-functional.
---
 govirt/ovirt-resource.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/govirt/ovirt-resource.c b/govirt/ovirt-resource.c
index 1413a77..0c750ac 100644
--- a/govirt/ovirt-resource.c
+++ b/govirt/ovirt-resource.c
@@ -499,7 +499,7 @@ G_GNUC_INTERNAL RestXmlNode *ovirt_resource_rest_call_sync(OvirtRestCall *call,
         return NULL;
     }
 
-    return root;
+    return ovirt_rest_xml_node_from_call(REST_PROXY_CALL(call));
 }
 
 
